docs/tutorial/advanced-autodiff.md

```diff
@@ -0,0 +1,3 @@
+# Advanced autodiff
```

Comment:
Nit: "automatic differentiation" (for those new to autodiff)

Reply:
Done